src/components/App/App.js

```diff
@@ -70,9 +70,19 @@ const App = () => {
     clearErrorMessage();
   };
 
+  const hideHowToBtn = (e) => {
+    const howTo = document.querySelector(".how-to");
+    howTo.classList.add("hidden");
+  }
+
+  const showHowToBtn = (e) => {
+    const howTo = document.querySelector(".how-to");
+    howTo.classList.remove("hidden");
+  }  
+
```

Comment:
Nice work, I wonder if there's a different way to do this? Either way it works and I like that :)

Reply:
I can look into that today if I get time. This probably isn't the most elegant solution.

Reply:
Like this one !